src/main/scala/codacy/plugins/test/multiple/MultipleTests.scala

```diff
@@ -22,9 +22,20 @@ object MultipleTests extends ITest {
 
   def run(docsDirectory: JFile, dockerImage: DockerImage, optArgs: Seq[String]): Boolean = {
     debug(s"Running MultipleTests:")
+    
+    val selectedTest = optArgs.sliding(2).collectFirst {
+      case Seq("--only", multipleTestDir) =>
+        multipleTestDir
+    }
     val multipleTestsDirectory = docsDirectory.toScala / DockerHelpers.multipleTestsDirectoryName
+
+    val directories = selectedTest match {
+      case Some(dirName) => Seq(multipleTestsDirectory / dirName)
```

Comment:
What if the directory doesn't exist? Will this return an explicit message?

Reply:
At the moment crashes with a java specific exception.
```
sbt:codacy-plugins-test> run multiple codacy-eslint --only not-existing
[info] running (fork) codacy.plugins.DockerTest multiple codacy-eslint --only not-existing
[error] [MultipleTests] Running MultipleTests:
[error] Exception in thread "main" java.io.FileNotFoundException: /var/folders/n_/dp3668hs6djb7rs2sk7twlf00000gn/T/6104179201222623396/docs/multiple-tests/not-existing/results.xml (No such file or directory)
```

Reply:
It's easy enough to understand what the problem is...